### DISCUSSION TITLE
Tracked Vehicles

### DISCUSSION THREAD

**Discussion by Covakel** [Courseplay/Courseplay_FS22#832] [Q&A]
I'm currently using a Fendt 1162MT to pull my 15.7m cultivator. Is there a way to limit the rate of turn of the tractor so it doesn't pinch (jackknife) the cultivator when it tries to turn. Every time it tries to make a sharp turn it gets stuck and stops. 

Is this possible, or does it just not work with these types of tractors?

**Comment by Tensuko** [comment]
I can check that and implement an adjustment for this Tractor.

**Reply by Covakel** [reply]
I'd imagine the issue would happen with all the tracked tractors, such as the John Deere and Challenger as well. 
Might be good to just add a Rate of Turn limit in the tractor options. 
Just an idea, I'll leave it with the experts ;)
